client/packages/host/src/Admin/ConfigurationSettings.tsx

```diff
@@ -63,6 +68,17 @@ export const ConfigurationSettings = () => {
           </BaseButton>
         }
       />
+      <Setting
+        title={t('label.configure-supply-level')}
+        component={
+          <BaseButton onClick={handleSupplyModalOpen}>
+            {t('label.edit')}
+          </BaseButton>
+        }
+      />
+      {isOpen && (
```

Comment:
You can set a toast here instead of going the permission checks at the modal level, if user clicks and don't have the correct permission -> don't open modal and toast to say they don't have permission to edit supply levels